webdriver-spec.html

```diff
@@ -5777,21 +5787,32 @@
  <li><p>Let <var>body</var> and <var>arguments</var>
   be <var>script arguments</var>’ data.
 
- <li><p>Let <var>result</var> be the result of calling <a>execute a function body</a>,
-  with arguments <var>body</var> and <var>arguments</var>.
+ <li><p>Let <var>promise</var> be <a>a new Promise</a>.
 
- <li><p>Wait for <a>execute a function body</a> to return
-  or for <a>session script timeout</a> milliseconds to expire,
-  whichever happens sooner.
-
- <li><p>If the previous step completed due to the <a>session script timeout</a> being reached,
-  return <a>error</a> with <a>error code</a> <a>script timeout</a>.
+ <li><p>Run the following substeps <a>in parallel</a>:
+   <ol>
+     <li><p>Let <var>result</var> be the result of <a>promise-calling</a>
+        <a>execute a function body</a>, with arguments
+        <var>body</var> and <var>arguments</var>.
 
- <li><p>If <var>result</var> is an <a>error</a>, return result.
+     <li><p>If <var>result</var> is an error,
+       <a>reject</a> <var>promise</var> with <var>result</var>.
+   </ol>
+   <li><p>Run the following substeps <a>in parallel</a>:
+     <ol>
+       <li><p>If <var>promise</var> is still pending and
```

Comment:
There's only one step here. Doesn't matter whether this is in parallel or in serial. Should this be part of the list above?